public/data/taravis.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Anu Wat",
+  "bio": "Web Dev, Interactive Media Dev",
```

Comment:
```suggestion
  "bio": "Web Dev | Interactive Media Dev",
```

Reply:
I think the content should be up to them unless it is a grammar/spelling mistake 🙂